backend/dependencies.py

```
@@ -0,0 +1,10 @@
+"""
+Dependency injection and shared dependencies for the FastAPI app.
+"""
+from fastapi import Depends, HTTPException
```

Ruff (0.11.9)

`fastapi.Depends` imported but unused

Remove unused import

(F401)